openjdk.test.load/config/inventories/mauve/mauve_all_exclude.xml

```diff
@@ -1,4 +1,11 @@
 <inventory>
+	<!-- Fails on JDK11 and up if run many times.
+		 Issue: https://github.com/adoptium/aqa-tests/issues/2293
+		 Note: The issue has been fixed upstream, and has been backported.
+		 We're waiting for the fix to move from JDKx-dev forks into JDKx.
```

Comment:
> We're waiting for the fix to move from JDKx-dev forks into JDKx

Is there any ETA on when this fix will become available and this exclude will be removed?

Reply:
I don't have an ETA, sorry.